src/components/DirectPaymentCreateModal/DirectPaymentCreateModal.tsx

```diff
@@ -286,142 +309,265 @@ export const DirectPaymentCreateModal: React.FC<DirectPaymentCreateModalProps> =
+                    onChange={handleInputChange}
+                    error={formErrors.walletId}
+                    disabled={isWalletFieldDisabled}
+                    note={getWalletFieldNote()}
+                    required
+                  >
+                    <option value="">Select wallet</option>
+                    {renderWalletOptions()}
+                  </Select>
+                </div>
               </div>
-            </div>
 
-            {/* External Payment ID */}
-            <div
-              className="DirectPaymentCreateModal__fieldWrapper"
-              data-visible={showReceiverField}
-            >
-              <div className="DirectPaymentCreateModal__fieldWrapper__content">
-                <Input
-                  fieldSize="sm"
-                  id="externalPaymentId"
-                  name="externalPaymentId"
-                  type="text"
-                  label="External Payment ID (optional)"
-                  placeholder="Enter external payment ID"
-                  value={formData.externalPaymentId}
-                  onChange={handleInputChange}
-                />
+              {/* External Payment ID */}
+              <div
+                className="DirectPaymentCreateModal__fieldWrapper"
+                data-visible={showReceiverField}
+              >
+                <div className="DirectPaymentCreateModal__fieldWrapper__content">
+                  <Input
+                    fieldSize="sm"
+                    id="externalPaymentId"
+                    name="externalPaymentId"
+                    type="text"
+                    infoText="An identifier for the payment defined by your organization"
+                    tooltipPlacement="top-start"
+                    label="Enter external payment ID (optional)"
+                    placeholder="Enter external payment ID"
+                    value={formData.externalPaymentId}
+                    onChange={handleInputChange}
+                  />
+                </div>
               </div>
             </div>
-          </div>
-        </Modal.Body>
-        <Modal.Footer>
-          <Button size="md" variant="tertiary" type="reset" disabled={isLoading}>
-            Cancel
-          </Button>
-          <Button
-            size="md"
-            variant="primary"
-            type="submit"
-            disabled={isLoading}
-            isLoading={isLoading}
-          >
-            Confirm payment
-          </Button>
-        </Modal.Footer>
-      </form>
+          </Modal.Body>
+          <Modal.Footer>
+            <Button size="sm" variant="secondary" type="reset" disabled={isLoading}>
+              Cancel
+            </Button>
+            <Button
+              size="sm"
+              variant="tertiary"
+              type="submit"
+              disabled={isLoading}
+              isLoading={isLoading}
+            >
+              Submit
+            </Button>
+          </Modal.Footer>
```

Comment:
I don't think these are the correct button colors. We usually have an action button purple (primary in new SDS) and a cancel button white (tertiary in new SDS). And size is larger ("md") as it was before.

<img width="747" height="489" alt="image" src="https://github.com/user-attachments/assets/b93cf2a9-58ed-43e4-b4a6-0e0f7aeb0e4e" />

Example from API key

<img width="747" height="129" alt="image" src="https://github.com/user-attachments/assets/83590a1e-7681-4154-b285-30e4f2f2c8a4" />